src/components/Range.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  label {
+    margin-right: 10px;
+  }
+  output {
+    background: white;
```

Comment:
I think it should come from the `theme`, probably `theme.input`?

Reply:
Done